src/sinks/splunk_hec/logs/config.rs

```diff
@@ -54,23 +64,31 @@ pub struct HecLogsSinkConfig {
     ///
     /// [splunk_field_index_docs]: https://docs.splunk.com/Documentation/Splunk/8.0.0/Data/IFXandHEC
     #[serde(default)]
+    #[configurable(metadata(docs::examples = "field1", docs::examples = "field2"))]
     pub indexed_fields: Vec<String>,
 
-    /// The name of the index where to send the events to.
+    /// The name of the index where to send events to.
     ///
-    /// If not specified, the default index is used.
+    /// If not specified, the default index defined within Splunk is used.
```

Comment:
👍 I definitely thought it was a default we set rather than on their side - if that's the case happy with this staying an option.